docs/site/examples.md

```diff
@@ -21,7 +29,7 @@ examples:
        url: "/tree/master/examples/python/tagging"
      - language: powercli
        url: "/tree/master/examples/powercli/tagging"
-     - language: golang
+     - language: go
```

Comment:
Assuming this was broken then in the old examples?

Reply:
It wasn't broken. The language is called `go` not `golang` and we only use `golang` as a search term to distinguish it from the word `go`.

But, good catch. I broke it by using `go`. I'll have that fixed.